src/components/Grid/SimpleGrid.stories.mdx

```diff
@@ -0,0 +1,346 @@
+
+The `SimpleGrid` component is used to render UI elements in a uniform grid layout, with the column widths and grid spacing fixed throughout the entire grid.
+
+<Canvas withToolbar>
+  <Story name="SimpleGrid Props">
+    {(args) => (
+      <SimpleGrid {...args}>
+        <Card
+          imageSrc="https://placeimg.com/500/200/animals"
+          imageAlt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        >
+          <CardHeading level={HeadingLevels.Three}>Card Heading</CardHeading>
+          <CardContent>
+            Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+            libero, a pharetra augue.
+          </CardContent>
+        </Card>
+        <Card
+          imageSrc="https://placeimg.com/400/220/animals"
+          imageAlt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        >
+          <CardHeading level={HeadingLevels.Three}>Card Heading</CardHeading>
+          <CardContent>
+            Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+            libero, a pharetra augue.
+          </CardContent>
+        </Card>
+        <Card
+          imageSrc="https://placeimg.com/400/240/animals"
+          imageAlt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        >
+          <CardHeading level={HeadingLevels.Three}>Card Heading</CardHeading>
+          <CardContent>
+            Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+            libero, a pharetra augue.
+          </CardContent>
+        </Card>
+        <Card
+          imageSrc="https://placeimg.com/400/260/animals"
+          imageAlt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        >
+          <CardHeading level={HeadingLevels.Three}>Card Heading</CardHeading>
+          <CardContent>
+            Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+            libero, a pharetra augue.
+          </CardContent>
+        </Card>
+        <Card
+          imageSrc="https://placeimg.com/400/320/animals"
+          imageAlt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        >
+          <CardHeading level={HeadingLevels.Three}>Card Heading</CardHeading>
+          <CardContent>
+            Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+            libero, a pharetra augue.
+          </CardContent>
+        </Card>
+        <Card
+          imageSrc="https://placeimg.com/400/300/animals"
+          imageAlt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        >
+          <CardHeading level={HeadingLevels.Three}>Card Heading</CardHeading>
+          <CardContent>
+            Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+            libero, a pharetra augue.
+          </CardContent>
+        </Card>
+      </SimpleGrid>
+    )}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Basic SimpleGrid" />
```

Comment:
This should be "SimpleGrid Props".

Reply:
Good catch.  A last minute change to the story name.